tests/test_generate_summary_outputs.py

```diff
@@ -117,7 +117,7 @@ def test_calculate_variant_rank_difference(self):
                     "Variant": "3-12563453454-C-T",
                     "/path/to/results_directory1": 9,
                     "/path/to/results_directory2": 3,
-                    "rank_decrease": -6,
+                    "rank_change": -6,
```

Comment:
Isn't a rank decrease of -6 equal to a rank change of 6?

Reply:
If we are comparing `results_directory1` to `results_directory2` using `results_directory1` as the baseline, hasn't the rank decreased by 6 ranks?